snappy/install.go

```diff
@@ -46,6 +46,8 @@ const (
 	AllowGadget
 	// DeveloperMode will install the snap without confinement
 	DeveloperMode
+	// TryMode indicates the snap is in try (unpacked directory) mode
+	TryMode
```

Comment:
It's unfortunate we need this here.

@pedronis: we should kill all of those flags and migrate the ones we really need into the existing flag set of snapstate.